wrapanapi/systems/ec2.py

```diff
@@ -506,6 +506,7 @@ def __init__(self, **kwargs):
         self.ssm_connection = boto3client('ssm', **connection_kwargs)
         self.sns_connection = boto3client('sns', **connection_kwargs)
         self.cw_events_connection = boto3client('events', **connection_kwargs)
+        self.resourcegroupstaggingapi_connection = boto3client('resourcegroupstaggingapi', **connection_kwargs)
```

Comment:
That's a heck of an endpoint name :laughing:

Reply:
btw not at all suggesting you change it - its consistent, and is exactly what it should be. It's just ... verbose.